src/ServiceControl.Monitoring/Retries/OccurrenceHandler.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace ServiceControl.Monitoring
+{
+    using System.Threading.Tasks;
+    using Metrics.Raw;
+    using NServiceBus;
+    public class OccurrenceHandler : IHandleMessages<Occurrences>
+    {
+        readonly RetriesStore store;
+        const string RetriesMessageType = "Retries";
+
+        public OccurrenceHandler(RetriesStore store)
+        {
+            this.store = store;
+        }
+
+        public Task Handle(Occurrences message, IMessageHandlerContext context)
+        {
+            try
+            {
+                var endpointInstanceId = EndpointInstanceId.From(context.MessageHeaders);
+
+                var messageType = context.MessageHeaders[MetricHeaders.MetricType];
+
+                switch (messageType)
+                {
+                    case RetriesMessageType:
+                        store.Store(endpointInstanceId, message.Entries);
+                        break;
+                    default:
+                        throw new UnknownOccurrenceMessageType(messageType);
```

Comment:
Should we have a custom retry policy so these messages get forwarded straight to the error queue instead of going through FLR+SLR?